blockly-ws-search/workspace_search.js

```diff
@@ -64,20 +71,6 @@ class WorkspaceSearch {
      * @type {boolean}
      */
     this.searchOnInput = true;
-
-    /**
-     * HTML container for the workspace search bar.
-     * @type {?HTMLElement}
-     * @private
-     */
-    this.HtmlDiv_ = null;
```

Comment:
Instead of getting rid of this, I would like to just update references in the code to be use this.HtmlDiv_ instead of HtmlDiv.

Reply:
ah, that's what happened. I've updated to use HtmlDiv_